common/language.c

```diff
@@ -1,4 +1,4 @@
-﻿#include "language.h"
+#include "language.h"
```

Comment:
What changed here? Make sure this isn't a stray whitespace change. Also make sure you didn't introduce an UTF-8 "BOM" by mistake (although if you *removed* it then that is *good*, please keep it that way then).

Reply:
Yes it lost some initial bytes (presumably the UTF-8 BOM) after I edited it. The # is now the first byte of the file.